packages/arianee-access-token/src/lib/arianee-access-token.ts

```diff
@@ -1,24 +1,31 @@
 import { Core } from '@arianee/core';
 import { MemoryStorage } from '@arianee/utils';
 import { ethers, JsonRpcProvider } from 'ethers';
+import memoizee from 'memoizee';
 
 import { JWTGeneric } from './helpers/jwtGeneric';
 import { ArianeeAccessTokenPayload } from './types/ArianeeAccessTokenPayload';
 import { JwtHeaderInterface } from './types/JwtHeaderInterface';
-
 export interface PayloadOverride {
   exp?: number;
   iat?: number;
   [key: string]: string | number | object | undefined;
 }
 
-const ETHEREUM_PROVIDER = 'https://ethereum.arianee.net/';
+const JSON_RPC_PROVIDER = new JsonRpcProvider(
+  'https://eth-mainnet.public.blastapi.io'
```

Comment:
Pq?

Reply:
way faster than our RPC, 75%+ on my tests

Reply:
On peut le variabiliser plutôt ? Car si ce noeud tombe, faut qu'on re-deploie une version du sdk, etc... pour résoudre le bug

Reply:
already the case 
```
  resolvedIss = await addressResolver(
          iss,
          options?.ethereumProvider ?? JSON_RPC_PROVIDER
        );
```

the constant is the default provider